### PR TITLE
Add sourceMappingURL to dest file.

### LINKED ISSUE #11: sourceMappingURL Directive Missing
Hi, I'm not sure if this issue needs to be raised here or in the main 6to5 repo.

I've set `sourceMap: true` in my gruntfile options, and the source maps are produced, but the sourceMappingURL directive is not added to the output JS files. `sourceMap: 'inline'` appears to add the directive correctly.

### PR DESCRIPTION
**Pull request by stebru**
Fixes #11

### CODE REVIEW

File: tasks/6to5.js [hunk 1]
```diff
@@ -11,7 +11,12 @@ module.exports = function (grunt) {
 
 			var res = to5.transformFileSync(el.src[0], options);
 
-			grunt.file.write(el.dest, res.code);
+			var sourceMappingURL = '';
+			if (res.map) {
+				sourceMappingURL = grunt.util.linefeed + '//# sourceMappingURL=' + el.dest.split('/').pop() + '.map';
```
Comment (sindresorhus) [inline tasks/6to5.js]: `'//# sourceMappingURL='` => `'\n//# sourceMappingURL='`, right?

Also, why `el.dest.split('/').pop()`?

**Comment by sindresorhus** [comment]
@sebmck Maybe the docs should say something about needing to add this to the source file? I know it's source map basic, but probably not apparent to most.

**Comment by sebmck** [comment]
@sindresorhus It'd probably be worth adding an option for 6to5 to do this. Right now `sourceMap: "inline"` can be used which will inline the sourcemap with a similar comment so maybe adding `sourceMap: "external"` and using the existing `sourceMapName` option.

**Comment by sindresorhus** [comment]
@sebmck It has to know the destination though, so you would need another option for that. I think it's easier to just tell people to add it.

File: tasks/6to5.js (hunk 1 above)
Comment (stebru) [inline tasks/6to5.js]: `grunt.util.linefeed == \n`, no?

The `el.dest` could be a path, but you only want the filename part for the sourceMappingURL directive. Seems uglify is doing it that way, anyway?

So:
```javascript
module.exports = {
    options: {
        sourceMap: true
    },
    dist: {
        files: {
            'some/path/6to5/dest.js': 'some/other/path/source.js'
        }
    }
};
```
ouputs `some/path/6to5/dest.map.js` and adds `//# sourceMappingURL=dest.js.map

Is this the wrong way to do it? Should it output the whole path?

Comment (sindresorhus) [inline tasks/6to5.js]: > grunt.util.linefeed == \n, no?

No, most tools output in `\n`, so just `\n`.

> The el.dest could be a path, but you only want the filename part for the sourceMappingURL directive. Seems uglify is doing it that way, anyway?

I guess. Source Maps escape me. But use `path.basename()` instead then.

Comment (stebru) [inline tasks/6to5.js]: Ah, I see, thx. Node noob here. Done.

**Comment by davemackintosh** [comment]
Will this be merged anytime soon?

**Comment by sebmck** [comment]
@sindresorhus Good to merge?

**Comment by sindresorhus** [comment]
Lgtm, but could use a test to make sure it's working correctly.

**Comment by stebru** [comment]
@sindresorhus Added test. Looks ok?

**Pull request closed without merging**